pypyr-ops/hooks/test

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash -xe
 
-EXPECTED_GO='go version go1.11.1'
-EXPECTED_PYPYR='pypyr 1.1.1'
-EXPECTED_TERRAFORM='Terraform v0.11.8'
-EXPECTED_AWS='aws-cli/1.16.37'
+EXPECTED_GO='go version go1.11.2'
+EXPECTED_PYPYR='pypyr 1.2.1'
+EXPECTED_TERRAFORM='Terraform v0.11.10'
+EXPECTED_AWS='aws-cli/1.16.48'
```

Comment:
add expected docker and kubectl tests.

Reply:
If I run `$(docker run --entrypoint docker ${IMAGE_NAME} version)` I get 
```
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running? 
```

Reply:
If I run `DOCKERVER=$(docker run -p --entrypoint kubectl ${IMAGE_NAME} version)`. I get
```
The connection to the server localhost:8080 was refused - did you specify the right host or port?
```

Reply:
doesn't it give something like this?
```
The connection to the server localhost:8080 was refused - did you specify the right host or port?
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.1", GitCommit:"4ed3216f3ec431b140b1d899130a69fc671678f4", GitTreeState:"clean", BuildDate:"2018-10-05T16:46:06Z", GoVersion:"go1.10.4", Compiler:"gc", Platform:"linux/amd64"}```

with exit code 1?

In which case, just trap the 1 return code, - the grep should still pick up "v1.12.1" from the client version info in there.

Reply:
`trap the 1 return code` how?

Reply:
I have trapped by using `command || true`